### ISSUE TITLE
Command Reference

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#21]
Particularly as we get more and more commands, it would be nice to have a way to see a reference sheet that would remind you the parameters required under each command while in-game.

Idea from @sam-irl was an autocomplete-esque thing where the text box would show in faint color the arguments available for the given command (such as typing `AAL123 turn` and after that it showing a faint `[L/R] [magnetic heading]` to clue people in on what the parmeters are for the `turn` command). 

Ideally, a full solution to this issue would include _both_ the above behavior in the text box as well as a list of commands that users can show/hide when they need to see what commands are available, what the differences are, etc.

Originally reported in zlsa/atc#463.

**Comment by n8rzz** [comment]
@erikquinn I suppose the wiki page wouldn't count as a command reference?

**Comment by erikquinn** [comment]
@n8rzz It counts as a thorough how-to, but sometimes a shortlist would be nice, for example, to remind oneself what command you use to "tell that plane to skip straight to WETOR" (`pd`) via a small in-game cheat sheet.

**Comment by ron-eo** [comment]
Is there a command that will bring your airport back to the center of the screen? I'm in empty space and feeling lost! :(

**Comment by indianbhaji** [comment]
Try double clicking an arriving flight on the flight strip - don't try this with a departure

**Comment by ron-eo** [comment]
Excellent! Thank you, indianbhaji! Something that would be useful in a command reference, that's for sure! Cheers ! :)

**Comment by scott-j-smith** [comment]
Actually, I do that with departures, but only after they are already waiting at the runway.

**Comment by n8rzz** [comment]
we did fix that bug this sprint (#207) that would break the app when double clicking on an aircraft strip for an aircraft that hasn't taxied.

**Comment by n8rzz** [comment]
@suhaildawood this story would be included with any sort of UI updating.  Just calling this one out for you because its old and you may not have seen it 😉
